roles/inspec/templates/etc/inspec/host-controls/attributes.yml

```diff
@@ -1,3 +1,6 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% for name,value in inspec.rhel7.attributes.iteritems() %}
```

Comment:
So, I am going to keep the structure of how the enablement is done the way it is. So role defaults enabled: True and the defaults2.0.yml enabled: False. This is consistent with what we did for serverspec. 
Added in checks where we do not putt down the inspec-stigs-rhel7 repo on Ubuntu